recipes/hrds/meta.yaml

```diff
@@ -0,0 +1,58 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 13 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "hrds" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/EnvModellingGroup/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 8dc291fbdec8517cecf3362a225c67229e58928edd0435f207715cf2dc912b17
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - scipy
+    - gdal
+    - pytest
```

Comment:
Is `pytest` really a runtime dependency?
```suggestion
```
It's not a problem if it is, just unusual.